src/Test.sol

```diff
@@ -1,734 +1,19 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.6.0 <0.9.0;
 
-import "./Script.sol";
-import "ds-test/test.sol";
-
-// Wrappers around Cheatcodes to avoid footguns
-abstract contract Test is DSTest, Script {
-    using stdStorage for StdStorage;
-
-    uint256 internal constant UINT256_MAX =
-        115792089237316195423570985008687907853269984665640564039457584007913129639935;
+import "./Assertions.sol";
+import "./Cheats.sol";
+import "./console.sol";
+import "./console2.sol";
+import "./Errors.sol";
+import "./Math.sol";
+import "./Storage.sol";
+import "./Utils.sol";
+
+abstract contract Test is Assertions, Cheats, Utils {
+    address internal constant VM_ADDRESS =
+        address(bytes20(uint160(uint256(keccak256("hevm cheat code")))));
```

Comment:
Same nit on one line + unnecessary cast that way all declarations are consistent